packages/chaire-lib-backend/src/api/uploads.socketRoutes.ts

```diff
@@ -89,23 +89,29 @@ export default function (
         }
     };
 
-    // TODO Handle file chunking
+    const getSavedFilePath = (options: FileUploadOptions) => {
+        // Get the folder where to save the file
+        const saveToFolder = allOptions.uploadDirs[options.uploadType] || allOptions.uploadDirs['imports'];
+        if (!allOptions.uploadDirs[options.uploadType]) {
+            console.error(`No folder set for upload type ${options.uploadType}. Saving to 'imports' directory.`);
+        }
+
+        const importFileName =
+            typeof options.data.filename === 'string' ? sanitizeFileName(options.data.filename) : 'upload.txt';
+
+        const saveToFile = `${saveToFolder}/${importFileName}`;
+
+        // Create the directory if it does not exist
+        fileManager.directoryManager.createDirectoryIfNotExistsAbsolute(saveToFolder);
```

Comment:
This is in a "getSavedFilePath" function.  A "get" function should not really have a side effect like this. It should be moved elsewhere.